src/scss/components/_logo.scss

```diff
@@ -0,0 +1,4 @@
+.logosection__logo {
```

Comment:
Jeśli jesteśmy w komponencie to dajemy tusz (zgodnie z BEM) tylko nazwę .logo

Reply:
done